persistence-modules/spring-jdbc/src/main/java/com/baeldung/spring/jdbc/template/testing/EmployeeDAO.java

```diff
@@ -20,4 +22,21 @@ public void setJdbcTemplate(JdbcTemplate jdbcTemplate) {
     public int getCountOfEmployees() {
         return jdbcTemplate.queryForObject("SELECT COUNT(*) FROM EMPLOYEE", Integer.class);
     }
+
+    public Employee getEmployeeById(int id) {
+        RowMapper<Employee> EmployeeRowMapper = (rs, rowNum) -> new Employee(rs.getInt("ID"), rs.getString("FIRST_NAME"), rs.getString("LAST_NAME"));
+
+        return jdbcTemplate.queryForObject("SELECT * FROM EMPLOYEE WHERE id=?", EmployeeRowMapper, id);
+    }
+
+    public Employee getEmployeeByIdV2(int id) {
+        RowMapper<Employee> EmployeeRowMapper = (rs, rowNum) -> new Employee(rs.getInt("ID"), rs.getString("FIRST_NAME"), rs.getString("LAST_NAME"));
```

Comment:
Local variables should start with a lower character.

Reply:
Good catch ;) done